datafusion/sqllogictest/src/engines/conversion.rs

```diff
@@ -41,11 +41,7 @@ pub(crate) fn varchar_to_str(value: &str) -> String {
 
 pub(crate) fn f16_to_str(value: f16) -> String {
     if value.is_nan() {
-        if value.is_sign_positive() {
-            "NaN".to_string()
-        } else {
-            "-NaN".to_string()
-        }
+        "NaN".to_string()
```

Comment:
Hmm, you mean on different platform, a same value could be positive and negative?

Reply:
If so, maybe it good to leave a comment here.

Reply:
Yes. a function can returns `-NaN` on x86 while 'NaN` on Arm.